alchemy/src/upstash/api.ts

```diff
@@ -1,4 +1,4 @@
-import type { Secret } from "../secret";
+import type { Secret } from "../secret.js";
```

Comment:
Unrelated to this PR, but I'd like to understand this better.

There's been weirdness in the erasable-types + node discussion, extensionless-imports, etc.

Is there a doc or some guidance I can learn from?